projects/core/src/occ/adapters/site-context/site-context.interceptor.ts

```diff
@@ -50,7 +49,11 @@ export class SiteContextInterceptor implements HttpInterceptor {
     request: HttpRequest<any>,
     next: HttpHandler
   ): Observable<HttpEvent<any>> {
-    if (request.url.includes(this.occEndpoints.getBaseUrl())) {
+    if (
+      this.activeCurr &&
+      this.activeLang &&
+      request.url.includes(this.occEndpoints.getBaseUrl())
```

Comment:
There is a slight change in behaviour.   If `activeCurr` is flasy, `activeLang` is not going to be used and vice versia.

Reply:
I change it to this: 
```
setParams: {
          lang: this.activeLang ?? '',
          curr: this.activeCurr ?? '',
        },
 ```